scripts/docker/CC-Devel/bin/install_npm.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
So is this `nvm` or `npm`? File name differs from mentions in the code.

Reply:
Done